themes/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "bootswatch",
+  "description": "Bootswatch is a collection of themes for Bootstrap.",
+  "version": "5.2.3",
+  "author": "Thomas Park",
+  "homepage": "https://bootswatch.com",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thomaspark/bootswatch.git"
+  },
+  "bugs": {
+    "url": "https://github.com/thomaspark/bootswatch/issues"
+  },
+  "scripts": {
+    "build": "grunt swatch",
+    "docs": "grunt docs",
+    "htmllint": "grunt htmllint",
+    "stylelint": "stylelint \"dist/**/*.scss\" --rd",
+    "release": "grunt release",
+    "test": "npm run stylelint"
+  },
+  "files": [
+    "dist/**/*.{css,scss}"
+  ],
+  "devDependencies": {
+    "@lodder/grunt-postcss": "^3.1.1",
+    "autoprefixer": "^10.4.12",
+    "bootstrap": "^5.2.3",
```

Comment:
The current codebase still relies on Bootstrap 4. This is probably the first change to do (the first comment to treat).